src/bindings/python/tests/test_utils/test_utils.py

```diff
@@ -4,10 +4,12 @@
 
 import pytest
 import os
+import numpy as np
+import openvino as ov
 from pathlib import Path
 from openvino.utils import deprecated, get_cmake_path
 from tests.utils.helpers import compare_models, get_relu_model
-
+from openvino.utils.postponed_constant import make_postponed_constant
```

Comment:
You may add `from openvino.utils.postponed_constant import make_postponed_constant` to this [`__init__`](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/utils/__init__.py), so for users it will be available `openvino.utils import make_postponed_constant`

Reply:
done